src/base.js

```diff
@@ -13,14 +13,25 @@
  * Removal or modification of this copyright notice is prohibited.
  *
  */
-import { readJSONSync, writeJSONSync } from '../../../src/utils/fs';
+import { Command, flags as flagParser } from '@oclif/command';
 
-export function theJSONIsRead() {
-	const { filePath } = this.test.ctx;
-	this.test.ctx.returnValue = readJSONSync(filePath);
-}
+export default class BaseCommand extends Command {
```

Comment:
Tests?

Reply:
The test is added on the other PR since it needs more functionality to be added.